webapp/platform/client/src/client4.ts

```diff
@@ -4314,6 +4315,14 @@ export default class Client4 {
             {method: 'post', body: JSON.stringify(body)},
         );
     };
+
+    // Schedule Post methods
+    createSchedulePost = (schedulePost: ScheduledPost, connectionId: string) => {
```

Comment:
Why are we passing the `connectionId` here?

Reply:
The API will use it to filter out the current client's WS connection when sending the WS event. This is to avoid duplicate data in store, as being done in some other APIs as well.

Reply:
That's what I was hoping since I've wanted this for so many other APIs 😀

Instead of passing it as an argument to this one method, what if we saved the `connectionId` on the `Client4` instance and then have that header automatically attached to all requests? It shouldn't hurt them, but it'd set us up to use this elsewhere pretty nicely while keeping `createSchedulePost` simpler

Reply:
Also, I just realized the name of this. It's got a small typo, so could you rename it to `createScheduledPost`

Reply:
Thats a nice idea, its something we can do in the fixathon.